ovn-tester/ovn_workload.py

```diff
@@ -147,6 +139,27 @@ def central_containers(self):
         return self.db_containers
 
 
+class RelayNode(Node):
+    def __init__(self, phys_node, container, mgmt_ip):
```

Comment:
This is slightly unrelated, but I was thinking whether there's a good way to improve semantics/logic of the `container` argument in the `Node` subclasses. All three `WorkerNode`, `CentralNode` and `RelayNode` accept `container` argument but it doesn't really affect anything. When creating these classes you have to match what's pretty much hardcoded. For the [relay](https://github.com/ovn-org/ovn-fake-multinode/blob/af8f4579bc487ad56bd7d7e024d627d9ea4a32a5/ovn_cluster.sh#L49) and [central](https://github.com/ovn-org/ovn-fake-multinode/blob/af8f4579bc487ad56bd7d7e024d627d9ea4a32a5/ovn_cluster.sh#L21), prefix for the container names is set in ovn-fake-multinode script and for the worker nodes it can be set via [prefix](https://github.com/ovn-org/ovn-heater/blob/6ad787025dd8027bb07185c4cca44c20e77ecb58/ovn-fake-multinode-utils/generate-hosts.py#L65) config option.

I think it'd be better if the logic of creating instances of these classes was entirely hidden from user and they'd be created automatically based on the values from test scenario and configs. Right now these classes give illusion of control but in reality user has to match names and counts defined somewhere else.

This is of course topic for entirely separate change, just something I was thinking about.

Reply:
That's a very good point.  I'll see if it's easy to handle with a new commit in this PR, otherwise I'll open an issue to follow up later, thanks!